### ISSUE TITLE
Dark mode for Wiki

### ISSUE THREAD

**Issue by LVRENT** [multitheftauto/mtasa-blue#2316]
**Describe the solution you'd like**
I don't know if this is the place to post a request like that, but as this is the development part of MTA, I will try here.
A dark mode for wiki. Not a theme. An on-off switch will be probably sufficient. There are some scripters (including me) that use the wiki at night and it just blows my eyes with that white theme. A nice feature to have at least.

**Describe alternatives you've considered**
Yeah, we can install browser themes to cover this but it just doesn't look right (some elements on the page are colored incorrectly, some are inversed, some are still white). I saw Pirulax developed a CSS theme but it was last updated 4 years ago, and the wiki interface was updated some time ago, more recently than the theme.

**Additional context**
I can try to help with styling if it's needed, but I don't know who to speak with to get this started.

**Comment by patrikjuvonen** [comment]
We have an existing beta theme available on Wiki named DarkVector.

You can find it by going to your preferences at https://wiki.multitheftauto.com/wiki/Special:Preferences#mw-prefsection-rendering

This theme is not complete, as we have a lot of inline styles across our Wiki which makes it very difficult to restyle all boxes. These issues will be remedied at some point when we get rid of inline styles in the Wiki pages.

**Comment by botder** [comment]
As a wiki guest you can test it with https://wiki.multitheftauto.com/wiki/Main_Page?useskin=DarkVector

**Comment by LVRENT** [comment]
Yeah, it's WIP as I can see, but it's acceptable for the moment. Thanks!

**Comment by Allerek** [comment]
I've wrote about this on Developers Discord, that we should pick one person to manage Wiki styles.
